packages/ui/src/i18n/dictionaries/authenticator/es.ts

```diff
@@ -3,31 +3,49 @@ export const esDict = {
     'La recuperación de la cuenta requiere información de contacto verificada',
   'Back to Sign In': 'Volver a inicio de sesión',
   'Change Password': 'Cambiar contraseña',
+  Changing: 'Cambiando',
   Code: 'Código',
   Confirm: 'Confirmar',
   'Confirm a Code': 'Confirmar un código',
   'Confirm Password': 'Confirmar contraseña',
   'Confirm Sign In': 'Confirmar inicio de sesión',
   'Confirm Sign Up': 'Confirmar registro',
+  'Confirm TOTP Code': 'Confirmar código TOTP',
   'Confirmation Code': 'Código de confirmación',
+  Confirming: 'Confirmando',
   'Create a new account': 'Crear una cuenta nueva',
+  'Creating Account': 'Creando cuenta',
   Email: 'Email',
   'Forgot Password': 'Olvidé mi contraseña',
   'Incorrect username or password': 'Nombre de usuario o contraseña incorrecta',
   'Invalid password format': 'Formato de contraseña inválido',
   'Invalid phone number format': 'Formato de número de teléfono inválido',
+  'Forgot your password?': '¿Olvidó su contraseña?',
+  'Hide password': 'Ocultar contraseña',
   'Loading...': 'Cargando...',
+  Username: 'Nombre de usuario',
   'New Password': 'Nueva contraseña',
   or: 'o',
   Password: 'Contraseña',
   'Phone Number': 'Número de teléfono',
   'Resend a Code': 'Reenviar un código',
+  'Reset your password': 'Restablecer su contraseña',
+  'Reset your Password': 'Restablecer su Contraseña',
   'Resend Code': 'Reenviar código',
   'Send Code': 'Enviar código',
+  Sending: 'Enviando',
+  'Setup TOTP': 'Configurar TOTP',
+  'Show password': 'Mostrar contraseña',
   'Sign In': 'Iniciar sesión',
+  'Sign In': 'Iniciar Sesión',
```

Comment:
This appears to be a duplicate of the translation above. Do we need both uppercase `Sign In` and lowercase `Sign in`?

Reply:
+1

Reply:
I base on the en.ts file

Reply:
Yes, but this is adding a second `Sign In` key. Did you mean to add a `Sign in` key here instead?